client/components/Category.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+
+const subjects = [
+  { name: 'Math' },
+  { name: 'English' },
+  { name: 'Science' },
+  { name: 'Computer' },
+];
+
+const useStyles = makeStyles((theme) => ({
+  catButton: {
+    padding: theme.spacing(3),
+    textAlign: 'center',
+    backgroundColor: theme.palette.primary.light,
+    color: theme.palette.primary.contrastText,
+    width: '100%',
+    '&:hover': {
+      backgroundColor: theme.palette.primary.dark,
+    },
+  },
+}));
+
```

Comment:
I like that you defined styles separately in each component.  Makes way more sense.